lineage/BaumWelch.py

```diff
@@ -88,9 +88,9 @@ def do_M_step(tHMMobj, MSD: list, betas: list, gammas: list):
         assert tHMMobj[0].fT is None
         T = do_M_T_step(tHMMobj, MSD, betas, gammas)
 
-        # all the objects in the population have the same T
-        for t in tHMMobj:
-            t.estimate.T = T
+        # each tHMMobj has its own T
+        for jj, t in enumerate(tHMMobj):
+            t.estimate.T = T[jj]
```

Comment:
This changes the transition matrix setup for both drugs. I thought we're trying to find support for sharing the transition matrix across the drug concentrations, as it is currently done?

Reply:
oh... I thought we do want to settle on having T separate for each condition.
Either way, we can use the figures I made for the response to reviewers. I will revert this code change.
So technically we don't want these changes... I will only keep the changes you made to fix that error.

Reply:
Should be done. The changes I kept are necessary (some are from simplifying `Results()` that I had not noticed need change in the other PR)
Should be ready to merge..